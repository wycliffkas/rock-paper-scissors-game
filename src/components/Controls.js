import React from "react";

const Controls = (props) => {
  return (
    <div className="control-buttons">
      <button type="button" className="weaponBtn" onClick={props.startGame}>
        Play
      </button>
      <button
        className="button-reset weaponBtn"
        type="button"
        onClick={props.resetGame}
      >
        Reset
      </button>
    </div>
  );
};

export default Controls;
