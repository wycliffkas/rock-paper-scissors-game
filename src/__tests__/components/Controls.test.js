import React from "react";
import { shallow } from "enzyme";
import Controls from "../../components/Controls";

describe("Controls", () => {
  const props = {
    resetGame: jest.fn(),
    startGame: jest.fn(),
  };

  const wrapper = shallow(<Controls {...props} />);
  it("should render without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
