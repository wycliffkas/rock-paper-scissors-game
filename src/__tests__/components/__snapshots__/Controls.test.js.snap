// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls should render without crashing 1`] = `ShallowWrapper {}`;
